src/kagglehub/clients.py

```diff
@@ -55,6 +56,17 @@ def __init__(self):
         self.credentials = get_kaggle_credentials()
         self.endpoint = get_kaggle_api_endpoint()
 
+    def _check_for_version_update(self, response):
```

Comment:
can you add the type hint to `response`. This should be: https://github.com/Kaggle/kagglehub/blob/ef7d0113ccd4890e9a75cbe9318c6907ecb554f9/src/kagglehub/clients.py#L142